### PR TITLE
VMI Cert:Add CRITICAL traces for debug

### PR DESCRIPTION
**Pull request by raviteja-b**
This commits add CRITICAL traces for debugging VMI certificate exchange issues

### CODE REVIEW

**Comment by raviteja-b** [comment]
This is to debug these kind of issues https://w3.rchland.ibm.com/projects/bestquest/?defect=SW545026

**Review by sunharis — APPROVED** [review]

File: include/ibm/management_console_rest.hpp [hunk 1]
```diff
@@ -953,6 +955,7 @@ inline void getCSREntryAck(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
             {
                 asyncResp->res.result(boost::beast::http::status::bad_request);
                 asyncResp->res.jsonValue["Description"] = "Bad CSR";
+                BMCWEB_LOG_ERROR << "SignCSR failed with Bad CSR";
```
Comment (gtmills) [inline include/ibm/management_console_rest.hpp:958]: This one is a ERROR but ones below are a INFO ? Why is this one a ERROR ?

Comment (raviteja-b) [reply include/ibm/management_console_rest.hpp:958]: this hits when user provides  invalid CSR

**Comment by gtmills** [comment]
PR message / Commit message need updated to reflect using Critical errors

**Review by gtmills — APPROVED** [review]

**Pull request merged**